spec/controllers/bibliographic_controller_spec.rb

```diff
@@ -230,4 +230,35 @@
       end
     end
   end
+
+  describe '#item_discharge' do
+    before do
+      stub_request(:post, /.*\.exlibrisgroup\.com\/almaws\/v1\/bibs\/.*\/holdings\/.*\/items\/.*/)
+        .with(headers: { 'Authorization' => 'apikey TEST_WR_KEY' })
+        .to_return(status: 200,
+                   headers: { "Content-Type" => "application/json" },
+                   body: Rails.root.join('spec', 'fixtures', 'files', 'alma', 'scan_23258767460006421.json'))
+    end
+
+    it "scans and returns the item, using the correct alma api key and re-setting the key after" do
```

Comment:
Can we add in parenthesis after `scans and returns`, (discharges).

Reply:
Yes but I was using "returns" as in "the endpoint returns the item data", not as in "the system returns / checks in the book" 😁 I just pushed an edit; let me know if it's more clear.